docs/content/preview/architecture/docdb-sharding/tablet-splitting.md

```diff
@@ -282,8 +312,8 @@ In the following example, a three-node cluster is created and uses a YCSB worklo
 The following known limitations are planned to be resolved in upcoming releases:
 
 * Colocated tables cannot be split. For details, see [#4463](https://github.com/yugabyte/yugabyte-db/issues/4463)
-* Tablet splitting should be disabled during an index backfill. For details, see [#6704](https://github.com/yugabyte/yugabyte-db/issues/6704)
-* Cross cluster replication currently does not work with tablet splitting. For details, see [#5373](https://github.com/yugabyte/yugabyte-db/issues/5373)
+* By default, tablet splitting is disabled by default for tables with cross cluster replication. It can be enabled by setting both [`enable_tablet_split_of_xcluster_replicated_tables`](../../../reference/configuration/yb-master/#enable-tablet-split-of-xcluster-replicated-tables) and [`enable_tablet_split_of_xcluster_bootstrapping_tables`](../../../reference/configuration/yb-master/#enable-tablet-split-of-xcluster-bootstrapping-tables) to `true`.
```

Comment:
`enable_tablet_split_of_xcluster_bootstrapping_tables` should be kept as false until https://github.com/yugabyte/yugabyte-db/issues/13170 gets landed, as xcluster bootstrapping + tablet splitting is currently unsupported